assets/app/vue/components/SelfServeForm.vue

```diff
@@ -57,7 +57,6 @@ const onDeleteAppPassword = async (evt) => {
 <template>
   <div class="form-container">
     <notice-bar type="error" v-if="errorText">{{ errorText }}</notice-bar>
-    <h2>App Passwords</h2>
     <p>Create App Passwords to login to mail clients like Thunderbird! You cannot view the password after creating it, so make sure to save it.</p>
```

Comment:
I notice when you delete an existing app password there is no confirm delete dialog; is this something that should be considered? I notice there is a confirm delete dialog when deleting the overall account which is awesome.

Reply:
Haha that's a good point. I'll add it in.